seed/static/seed/js/services/inventory_service.js

```diff
@@ -498,6 +498,37 @@ angular.module('BE.seed.service.inventory', []).factory('inventory_service', [
       localStorage.setItem('detailProfiles.' + key, JSON.stringify(profiles));
     };
 
+    inventory_service.get_property_column_names_for_org = function (org_id) {
+      return $http.get('/api/v3/columns/', {
+        params: {
+          inventory_type: 'property',
+          organization_id: org_id,
+          only_used: false,
+          display_units: true
+        }
+      }).then(function (response) {
+        let property_columns = response.data.columns.filter(column => column.table_name == 'PropertyState');
+        return property_columns.map(a => {
+          return { 'column_name': a.column_name, 'display_name': property_columns.find(x => x.column_name == a.column_name).display_name };
+        });
+      });
+    };
+
+    inventory_service.get_taxlot_column_names_for_org = function (org_id) {
```

Comment:
I didn't realize this before, but these are very similar to the `get_property_columns`/`get_property_columns_for_org` (same for taxlot) functions, not sure what the best option is here. I guess we could modify the original func with `ui_grid_formatting = true` then conditionally format the display name? Or are there other differences I'm missing as well?

Reply:
This endpoint is returning an array of tuples, basically, with only the column name and the display name.  This tuple array is used to populate the select from the org settings page.  Do you think the original functions could and should be used for this?

Reply:
I'm good with just keeping it how you have it now, easy enough and your functions are simple.